manifests/00-crd.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: consoles.console.openshift.io
+  name: consoleoperatorconfigs.console.operator.openshift.io
```

Comment:
@spadgett fyi updating the CRD to conform with the most current patterns.

Reply:
`consoleoperatorconfigs` is what this represents now. `console.operator.openshift.io` is our group. This follows the pattern from [kube api server operator](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/manifests/0000_10_kube-apiserver-operator_01_config.crd.yaml#L4).